internal/cmd/givc-agent/main.go

```diff
@@ -90,6 +91,16 @@ func main() {
 	if wifiOption && (wifiService != "false") {
 		wifiEnabled = true
 	}
+
+	hwidEnabled := false
+	hwidService, hwidOption := os.LookupEnv("HWID")
+	hwidIface, hwidIfOption := os.LookupEnv("WIFI")
```

Comment:
Needs to read `HWID_IFACE`. Also, I noticed the problem that on my machine the interface is called `wlp0s6f0`, instead of `wlp0s5f0` (which is the name in the hw config). So if I set the interface like 
`hwidIface = (lib.head config.ghaf.hardware.definition.network.pciDevices).name;` it won't work.

Reply:
Yes, this was a copy-pasta error. The wrong interface name is a more problematic issue; the naming should be consistent, but I guess there may be some hardware revision change or something.

Could also look for the interface through enumeration, but if we ever need to support more than one adapter, this may become problematic.

Reply:
We shouldn't need to support multiple adapters though, as this is only a temporary solution. Long term we need to get a proper hardware attestation including ID. How about we use wlp0* for now?

Reply:
I think that would be `wl*`; if the cloud-hypervisor for net-vm PR gets merged, the interface is `wls8`.